cypress/e2e/tests/migration/applicationinventory/assessment/create.test.ts

```diff
@@ -105,6 +106,16 @@ describe(["@tier2"], "Application validations", () => {
         inputText(applicationNameInput, data.getFullName());
         cy.get(commonView.submitButton).should("not.be.disabled");
 
+        // Owner validation, Polarian TC 330
+        inputText(applicationOwnerInput, stakeHoldersList[0].name);
+        cy.get("button").contains(stakeHoldersList[0].name).click();
+
+        cy.get(applicationOwnerInput)
+            .invoke("val")
+            .then((value) => {
+                expect(value).to.contain(stakeHoldersList[0].name);
+            });
+
```

Comment:
Could you add one more validation?

Once added, clicking on the X button and verifying the owner name is not in the input anymore

Reply:
Sure!

Reply:
Updated the code with relevant changes, can you take a look? @sshveta @abrugaro 
Thanks!